chia/full_node/block_store.py

```diff
@@ -400,6 +406,37 @@ async def get_block_records_by_hash(self, header_hashes: List[bytes32]):
             ret.append(all_blocks[hh])
         return ret
 
+    async def get_block_blobs_by_hash(self, header_hashes: List[bytes32]) -> List[bytes]:
```

Comment:
The current naming scheme is this:

| return type | single hash | multiple hashes |
| --- | --- | --- |
| `FullBlock` | `get_full_block()` | `get_blocks_by_hash()` |
| `bytes` | `get_full_block_bytes()` | *this is your new function* |

I think the naming would be more consistent if this function would be called `get_block_bytes_by_hash()`.

Reply:
Also, this function needs test coverage in the block store test